fastparse/shared/src/test/scala/fastparse/ByteTests.scala

```diff
@@ -266,37 +267,579 @@ object ByteTests extends TestSuite {
       }
     }
     'example{
+      'bs{
+        import fastparse.byte._
+
+        assert(BS(0x01, 0xff).toSeq == Array[Byte](0x01.toByte, 0xff.toByte).toSeq)
+        assert(BS("hello":_*).toSeq == Array[Byte](104, 101, 108, 108, 111).toSeq)
+      }
+      'hexBytes{
+        import fastparse.byte._
+
+        assert(hexBytes("01 ff").toSeq == Array[Byte](0x01.toByte, 0xff.toByte).toSeq)
+      }
+      'prettyBytesBare{
+        import fastparse.byte._
+        val blob = BS(
+          "The quick brown fox jumps over the lazy dog. She sells " +
+          "sea shells on the sea shore but the shells she sells she " +
+          "sells no more. Peter piper picked a pack of pickled peppers.":_*
+        )
+        assert( prettyBytes(blob) ==
+          """      0  1  2  3  4  5  6  7  8  9  10 11 12 13 14 15
+            |
+            |0     54 68 65 20 71 75 69 63 6b 20 62 72 6f 77 6e 20
+            |16    66 6f 78 20 6a 75 6d 70 73 20 6f 76 65 72 20 74
+            |32    68 65 20 6c 61 7a 79 20 64 6f 67 2e 20 53 68 65
+            |48    20 73 65 6c 6c 73 20 73 65 61 20 73 68 65 6c 6c
+            |64    73 20 6f 6e 20 74 68 65 20 73 65 61 20 73 68 6f
+            |80    72 65 20 62 75 74 20 74 68 65 20 73 68 65 6c 6c
+            |96    73 20 73 68 65 20 73 65 6c 6c 73 20 73 68 65 20
+            |112   73 65 6c 6c 73 20 6e 6f 20 6d 6f 72 65 2e 20 50
```

Comment:
Yeah, okay. I don't think it's very important right now, but as you start this thing with byte pretty formatting, you can make it more rich and with various options.